### PR TITLE
Describe MPR (v2) description

### PR DESCRIPTION
**Pull request by tknippenberg**

### CODE REVIEW

File: content/en/docs/refguide/version-control/version-control-troubleshooting/repository-size.md [hunk 1]
```diff
@@ -36,6 +36,21 @@ The *.mpr* storage format will be changed to reduce the rapid repository growth.
...
 
+#### 4.1.1 MPR format
+
+An app modeled in Mendix is traditionally stored in a single *.mpr* file. This is essentially a database which contains the data for all documents, such as microflows and pages. As the Mendix app is stored in a single file, your version control system will only see that a single file is changed. To show which documents have exactly changed inside the *.mpr* file a tool that comprehends the format is required, such as Studio Pro.
+
+#### 4.1.2 Repository growth
+
+Version control systems such as Git do not store a full copy of a document for every commit. Instead, they store the difference between the two revisions, also called the delta. In the case of binary files, such as the *.mpr* file, Git can't effectively calculate a delta. Where a small delta of a couple kilobytes is to be expected when a microflow is changed, the storage format can result in a delta of a megabyte or more. The consequence of this is that your Git repository grows more rapidly than you would expect based on the changes made.
+
+#### 4.1.3 Format change
```
Comment (MariaShaposhnikova) [inline content/en/docs/refguide/version-control/version-control-troubleshooting/repository-size.md]: Future release

Comment (tknippenberg) [reply content/en/docs/refguide/version-control/version-control-troubleshooting/repository-size.md]: Text is now version-independent and can be released as such.

**Comment by tknippenberg** [comment]
@MariaShaposhnikova , I've went through the PR with @mikita-haurykau and made the necessary technical corrections. 

I also rephrased the PR so it mentions this format will be introduced in a future version. That allows us to publish it now already, which is something I would like, so I can refer to it from the next email round that's coming up. 

I hope you can do the normal text-review, but aside from that it's ready to go.

File: content/en/docs/refguide/version-control/version-control-troubleshooting/repository-size.md (hunk 1 above)
Comment (MariaShaposhnikova) [reply content/en/docs/refguide/version-control/version-control-troubleshooting/repository-size.md]: @tknippenberg , I am still now sure we should publish a section that fully describes a functionality that is not yet there. Documentation usually publishes something that is available to all customers.

**Review by tknippenberg — COMMENTED** [review]
LGTM! Ready for publication from my end.

**Pull request merged**